mkt/inapp/tests/test_views.py

```diff
@@ -201,6 +204,37 @@ def test_filter(self):
         eq_(response.json['objects'][0]['guid'], inactive.guid)
 
 
+class TestInAppProductsWithPackagedWebApp(AuthenticatedInAppProductTest):
+
+    def setUp(self):
+        super(TestInAppProductsWithPackagedWebApp, self).setUp()
+        # Set up a packaged web app, i.e. one without a declared domain.
+        self.webapp.update(app_domain=None, is_packaged=True)
+        self.product = self.create_product()
+
+    def marketplace_origin(self):
+        return 'marketplace:{}'.format(self.webapp.guid)
```

Comment:
I know it's a helpers but we don't want all these tests to run with a webapp that has no guid since that might mess things up. Settings one explicitly in `setUp` might work too.